### PR TITLE
DP Vector Mean Accumulator

### PR DESCRIPTION
**Pull request by levzlotnik**
## Description
Vector Mean Accumulator implementation. Continuation for the work on Vector Sum Accumulator.

## Affected Dependencies
None.

## How has this been tested?
WIP

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by levzlotnik** [comment]
@dvadym 
Implemented Vector Mean Accumulator by computing the DP Vector Sum + Count and then dividing the two. Is there a better way of doing that?

File: pipeline_dp/dp_computations.py [hunk 1]
```diff
@@ -149,6 +149,12 @@ class AdditiveVectorNoiseParams:
     noise_kind: pipeline_dp.NoiseKind
 
 
+@dataclass
+class VectorMeanVarParams:
+    params_add: AdditiveVectorNoiseParams
```
Comment (dvadym) [inline pipeline_dp/dp_computations.py:154]: Nit: 
sum_params
count_params

File: pipeline_dp/accumulator.py [hunk 2]
```diff
@@ -239,6 +239,54 @@ def compute_metrics(self):
         return dp_computations.add_noise_vector(self._vec_sum, self._params)
 
 
+class VectorMeanAccumulator(Accumulator):
+    _vec_sum: np.ndarray
```
Comment (dvadym) [inline pipeline_dp/accumulator.py:243]: I missed this in the previous CL, this means that _vec_sum/_params are class level variables and in __init__ instance level _vec_sum/_params created. Please remove (and in `VectorSummationAccumulator` as well)

**Review by dvadym — COMMENTED** [review]
Thanks, it looks good in general. I left some comments.

File: pipeline_dp/accumulator.py (hunk 2 above)
Comment (levzlotnik) [reply pipeline_dp/accumulator.py]: Agreed, done.

File: pipeline_dp/dp_computations.py (hunk 1 above)
Comment (levzlotnik) [reply pipeline_dp/dp_computations.py]: Agreed, done.

**Comment by dvadym** [comment]
The code has changed significantly, this PR is obsolete

**Pull request closed without merging**